packages/acq/src/lib/api/api.spec.ts

```diff
@@ -53,22 +53,55 @@ describe('./data', () => {
     ])('should get the ledger data from the storage of a contract', async (storage, expected) => {
       // when ... we want the ledger from the storage of a contract
       // then ... it should fetch and format as expected
-      const httpStub = { get: jest.fn().mockResolvedValue(storage) }
-      const result = await SUT._getOwnedAssetsForPKH(httpStub as any)({
+      const httpStub = jest.fn().mockResolvedValue(storage)
+      const result = await SUT._getOwnedAssetsForPKH(httpStub as any)()({
         network: Network.ghostnet,
         contract: 'CONTRACT',
         contractType: AssetContractType.multi,
         pkh: validPkh,
       })
-
+      expect(httpStub).toHaveBeenCalledWith(
+        'https://api.ghostnet.tzkt.io/v1/contracts/CONTRACT/bigmaps/ledger/keys?key.address=tz1L9r8mWmRPndRhuvMCWESLGSVeFzQ9NAWx&value.gt=0',
```

Comment:
Is it fine to write the address explicitly everywhere or should you create a CONST somewhere?